src/hooks/useResponsiveTime.ts

```diff
@@ -0,0 +1,13 @@
+import { useState, useEffect } from 'react';
```

Comment:
I don't think we need `useState` and `useEffect` here. Instead of the hook, you could call the `setConversationTime` function from the components. For example instead of 

```TypeScript
const responsiveTime = useResponsiveTime(topMessageTime);
...
<div className="ConversationListItem Time">{responsiveTime}</div>
```
you would just have
```TypeScript
<div className="ConversationListItem Time">{setConversationtime(topMessageTime)}</div>
```

Reply:
I removed the hook format, imported the function and adjusted the code with the new push